CONTRIBUTING.md

```diff
@@ -0,0 +1,219 @@
+All contributions to this project will be released under the CC0 public domain
+dedication. By submitting a pull request or filing a bug, issue, or
+feature request, you are agreeing to comply with this waiver of copyright interest.
+Details can be found in our [TERMS](https://github.com/cfpb/design-system/blob/master/TERMS.md) and [LICENSE](https://github.com/cfpb/design-system/blob/master/LICENSE).
```

Comment:
TERMS link is broken, should link to a version of this file https://github.com/cfpb/open-source-project-template/blob/master/TERMS.md
LICENSE is wrong, says MIT, should be our own custom license from https://github.com/cfpb/open-source-project-template/blob/master/LICENSE

Reply:
These links should resolve once these changes reach master.